### PR TITLE
feat(webvitals): Adds docs for enableInp option

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Adds documentation for the `enableInp` option, under the javascript auto instrumentation page

image

### CODE REVIEW

**Review by narsaynorath — APPROVED** [review]

File: docs/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 1]
```diff
@@ -130,3 +130,16 @@ The default is `true`.
 This option determines whether spans for long tasks automatically get created.
 
 The default is `true`.
+
+
+### enableInp
+
+<Note>
+The `enableInp` option requires SDK version [version 7.104.0](https://github.com/getsentry/sentry-javascript/releases/tag/7.104.0) or higher.
```
Comment (vivianyentran) [inline docs/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx:138]:
```suggestion
The `enableInp` option requires SDK [version 7.104.0](https://github.com/getsentry/sentry-javascript/releases/tag/7.104.0) or higher.
```

File: docs/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 2]
```diff
@@ -130,3 +130,16 @@ The default is `true`.
...
+</Note>
+
+This option determines whether interactions spans automaticallly get created when an [Interaction to Next Paint (INP)](/product/performance/web-vitals/web-vitals-concepts/#interaction-to-next-paint-inp) event is detected. Interactions are scored and surfaced in the [Web Vitals](/product/performance/web-vitals/) module.
+
+the default is `false`.
```
Comment (vivianyentran) [inline docs/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx:143]:
```suggestion
The default is `false`.
```

File: docs/product/performance/web-vitals/index.mdx [hunk 3]
```diff
@@ -5,7 +5,7 @@ description: "Understand your application's performance score and how each web v
...
+On March 12, 2024, Google replaced [First Input Display (FID)](/product/performance/web-vitals/web-vitals-concepts/#first-input-delay-fid) with [Interaction to Next Paint (INP)](/product/performance/web-vitals/web-vitals-concepts/#interaction-to-next-paint-inp) as a Core Web Vital. To begin collecting INP measurements, make sure your Javascript SDK version is [7.104.0](https://github.com/getsentry/sentry-javascript/releases/tag/7.104.0) or higher and that the option [enableInp](/platforms/javascript/performance/instrumentation/automatic-instrumentation/#enableinp) is on. FID measurements are no longer shown in the **Web Vitals** page, but can still be accessed in [Discover](/product/discover-queries). You may see changes to your performance score during this transition.
```
Comment (vivianyentran) [inline docs/product/performance/web-vitals/index.mdx:8]:
```suggestion
On March 12, 2024, Google replaced [First Input Display (FID)](/product/performance/web-vitals/web-vitals-concepts/#first-input-delay-fid) with [Interaction to Next Paint (INP)](/product/performance/web-vitals/web-vitals-concepts/#interaction-to-next-paint-inp) as a Core Web Vital. To begin collecting INP measurements, make sure your Javascript SDK version is [7.104.0](https://github.com/getsentry/sentry-javascript/releases/tag/7.104.0) or higher and that the option [`enableInp`](/platforms/javascript/performance/instrumentation/automatic-instrumentation/#enableinp) is on. FID measurements are no longer shown in the **Web Vitals** page, but can still be accessed in [Discover](/product/discover-queries). You may see changes to your performance score during this transition.
```

File: docs/product/performance/web-vitals/web-vitals-concepts.mdx [hunk 4]
```diff
@@ -7,7 +7,7 @@ description: "Understand the health of your application with Web Vitals by monit
...
 
 <Note>
-On March 12, 2024, Google replaced [First Input Display (FID)](#first-input-delay-fid) with [Interaction to Next Paint (INP)](#interaction-to-next-paint-inp) as a Core Web Vital. To begin collecting INP measurements, make sure your Javascript SDK version is [7.104.0](https://github.com/getsentry/sentry-javascript/releases/tag/7.104.0) or higher and that the option `enableInp` is on. FID measurements are no longer shown in the **Web Vitals** page, but can still be accessed in [Discover](/product/discover-queries).
+On March 12, 2024, Google replaced [First Input Display (FID)](#first-input-delay-fid) with [Interaction to Next Paint (INP)](#interaction-to-next-paint-inp) as a Core Web Vital. To begin collecting INP measurements, make sure your Javascript SDK version is [7.104.0](https://github.com/getsentry/sentry-javascript/releases/tag/7.104.0) or higher and that the option [enableInp](/platforms/javascript/performance/instrumentation/automatic-instrumentation/#enableinp) is on. FID measurements are no longer shown in the **Web Vitals** page, but can still be accessed in [Discover](/product/discover-queries).
```
Comment (vivianyentran) [inline docs/product/performance/web-vitals/web-vitals-concepts.mdx:10]:
```suggestion
On March 12, 2024, Google replaced [First Input Display (FID)](#first-input-delay-fid) with [Interaction to Next Paint (INP)](#interaction-to-next-paint-inp) as a Core Web Vital. To begin collecting INP measurements, make sure your Javascript SDK version is [7.104.0](https://github.com/getsentry/sentry-javascript/releases/tag/7.104.0) or higher and that the option [`enableInp`](/platforms/javascript/performance/instrumentation/automatic-instrumentation/#enableinp) is on. FID measurements are no longer shown in the **Web Vitals** page, but can still be accessed in [Discover](/product/discover-queries).
```

File: docs/product/performance/web-vitals/web-vitals-concepts.mdx [hunk 5]
```diff
@@ -25,7 +25,7 @@ Google considers Core Web Vitals to be the most important metrics for measuring
 ### Interaction to Next Paint (INP)
 
 <Note>
-On March 12, 2024, Interaction to Next Paint (INP) replaced First Input Display (FID) as a Core Web Vital. Prior to this, INP was an experimental metric that Sentry did not collect. To begin collecting INP measurements, make sure your Javascript SDK version is [7.104.0](https://github.com/getsentry/sentry-javascript/releases/tag/7.104.0) or higher and that the option `enableInp` is on.
+On March 12, 2024, Interaction to Next Paint (INP) replaced First Input Display (FID) as a Core Web Vital. Prior to this, INP was an experimental metric that Sentry did not collect. To begin collecting INP measurements, make sure your Javascript SDK version is [7.104.0](https://github.com/getsentry/sentry-javascript/releases/tag/7.104.0) or higher and that the option [enableInp](/platforms/javascript/performance/instrumentation/automatic-instrumentation/#enableinp) is on.
```
Comment (vivianyentran) [inline docs/product/performance/web-vitals/web-vitals-concepts.mdx:28]:
```suggestion
On March 12, 2024, Interaction to Next Paint (INP) replaced First Input Display (FID) as a Core Web Vital. Prior to this, INP was an experimental metric that Sentry did not collect. To begin collecting INP measurements, make sure your Javascript SDK version is [7.104.0](https://github.com/getsentry/sentry-javascript/releases/tag/7.104.0) or higher and that the option [`enableInp`](/platforms/javascript/performance/instrumentation/automatic-instrumentation/#enableinp) is on.
```

**Review by vivianyentran — APPROVED** [review]
Added some tiny edits but otherwise LGTM! Thanks for adding this!

**Pull request merged**